docs/getting_started/quickstart.md

```diff
@@ -0,0 +1,58 @@
+# Quickstart
+
+This page explains how to quickly install and set up eZ Platform for development.
+
+## Basic requirements
+
+The recommended setup for using eZ Platform is:
+
+- Linux operating system (Debian, Ubuntu or RHEL / CentOS)
+- PHP 7.2
+    - with following packages: `php-cli`, `php-fpm`, `php-mysql`, `php-xml`, `php-mbstring`, `php-intl`, `php-curl`, `php-gd`
+- MySQL
+
+For more details see [Full requirements](requirements_and_system_configuration.md).
+
+## Installation
+
+To install eZ Platform you need [Composer](https://getcomposer.org/) and git on your system.
+
+``` bash
+composer create-project --keep-vcs ezsystems/ezplatform .
+composer ezplatform-install
+```
+
+For more details see [Install using Composer](install_using_composer.md).
+
+See [Install manually](install_manually.md) to install eZ Platform on a different OS.
+
+!!! tip "Demo site"
```

Comment:
As this page explains how to quickly install and set up eZ Platform for development do we want to mention demo here? 
If you want to keep it I would move the note to the top, just under the introduction.